bin/cylc-unregister

```diff
@@ -56,55 +58,43 @@ def main():
 
     db = RegistrationDB(options.db)
 
-    dirs = db.unregister(args[0])
-
-    n = len(dirs)
-    if n == 0:
-        print 'No suites unregistered.'
-        sys.exit(0)
-
-    print len(dirs), 'suites unregistered.'
-
-    if options.obliterate and len(dirs) > 0:
-        for dir in dirs:
-            print 'DELETE ', dir
-
-        really_obliterate = False
-        if options.force:
-            really_obliterate = True
-        else:
-            if len(dirs) == 1:
-                words = "THIS SUITE DEFINITION"
+    unregistered_list, skipped_list = db.unregister(args[0])
+
+    print '%d suite(s) unregistered.' % len(unregistered_list)
+    if not unregistered_list or not options.obliterate:
+        sys.exit(bool(skipped_list))
+
+    for _, path in unregistered_list:
+        if not options.force:
+            response = raw_input('REALLY DELETE "%s"? (y/n/a) ' % (path))
+            if response == 'a':
+                options.force = True
+            elif response != 'y':
+                continue
+        print 'DELETE "%s"' % (path)
+        try:
+            rmtree(path)
+        except OSError as exc:
+            print >> sys.stderr, (
+                "ERROR, could not remove directory: " + path)
+            print >> sys.stderr, str(exc)
+            continue
+        # recursively remove empty superdirs
+        tmp = path
+        while True:
+            tmp = os.path.split(tmp)[0]
+            try:
+                os.rmdir(tmp)
+            except OSError:
+                break
             else:
```

Comment:
This block (in master too) is a bit too simplistic to do what it's supposed to - i.e. remove all empty directories left by unregistering and deleting a bunch of suites.  To see this, try `cylc import examples $TMPDIR` then `cylc unreg -d cylc-.*` and look in `$TMPDIR`.  However, I doubt that anyone stores suites in directory trees that reflect the suite name hierarchy as I originally intended (e.g. `nwp.test1` and `nwp.test2` stored in `nwp/test1/` and `nwp/test2`) - certainly Rose users don't - and unregisters them en-masse ... so maybe we should just remove this feature.